libs/application/templates/driving-instructor-registrations/src/forms/instructorRegistrations.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  buildForm,
+  buildSection,
+  Form,
+  FormModes,
+  buildExternalDataProvider,
+  buildDataProviderItem,
+  buildMultiField,
+  buildCustomField,
+} from '@island.is/application/core'
+import { m } from '../lib/messages'
+
+export const getInstructorRegistrations = (): Form => {
+  return buildForm({
+    id: 'InstructorRegistrationsTemplate',
+    title: '',
+    mode: FormModes.APPLYING,
+    renderLastScreenButton: false,
+    children: [
+      buildSection({
+        id: 'externalData',
+        title: m.dataCollectionTitle,
+        children: [
+          buildExternalDataProvider({
+            id: 'approveExternalData',
+            title: m.dataCollectionTitle,
+            subTitle: m.dataCollectionSubtitle,
+            checkboxLabel: m.dataCollectionCheckboxLabel,
+            dataProviders: [
+              /*buildDataProviderItem({
```

Comment:
this should probably not be commented out, right?

Reply:
ó yea